projects/angular/src/forms/combobox/_combobox.clarity.scss

```
@@ -50,132 +34,59 @@
-      variables.$clr-use-custom-properties
-    );
+    border-bottom-left-radius: combobox-variables.$clr-combobox-border-radius;
+    border-bottom-right-radius: combobox-variables.$clr-combobox-border-radius;
   }
   .clr-focus {
     width: 100%;
   }
   .clr-error .clr-focus-indicator {
-    @include mixins.css-var(
-      background-color,
-      clr-forms-invalid-color,
-      forms-variables.$clr-forms-invalid-color,
-      variables.$clr-use-custom-properties
-    );
+    background-color: forms-variables.$clr-forms-invalid-color;
   }
 
   .clr-combobox-wrapper {
     position: relative;
 
     //Dimensions
-    padding: 0 caret-gap() 0 combobox-variables.$clr-combobox-trigger-left-padding;
-    @include mixins.css-var(
-      padding-left,
-      clr-combobox-padding-left,
-      combobox-variables.$clr-combobox-trigger-left-padding,
-      variables.$clr-use-custom-properties
-    );
-
-    @include mixins.css-var(
-      min-height,
-      clr-combobox-min-height,
-      combobox-variables.$clr-combobox-min-height,
-      variables.$clr-use-custom-properties
-    );
-    @include mixins.css-var(
-      min-width,
-      clr-combobox-min-width,
-      combobox-variables.$clr-combobox-min-width,
-      variables.$clr-use-custom-properties
-    );
+    padding: 0 caret-gap() 0 combobox-variables.$clr-combobox-trigger-padding-left;
+    padding-left: combobox-variables.$clr-combobox-padding-left;
+
+    min-height: combobox-variables.$clr-combobox-min-height;
+    min-width: combobox-variables.$clr-combobox-min-width;
     .clr-input {
-      padding-right: 0;
-      @include mixins.css-var(
-        padding-left,
-        clr-combobox-input-left,
-        variables.$clr_baselineRem_0_25,
-        variables.$clr-use-custom-properties
-      );
-      // ATM remove
-      @include mixins.css-var(
-        max-height,
-        clr-combobox-input-height,
-        variables.$clr_baselineRem_1_25,
-        variables.$clr-use-custom-properties
-      );
+      padding: 0;
+      max-height: tokens.$cds-global-space-7;
+
       &:not(:disabled) {
-        @include mixins.css-var(color, clr-combobox-text-color, inherit, variables.$clr-use-custom-properties);
+        color: combobox-variables.$clr-combobox-text-color;
       }
     }
-    @include mixins.css-var(
-      font-size,
-      clr-combobox-font-size,
-      combobox-variables.$clr-combobox-font-size,
-      variables.$clr-use-custom-properties
-    );
-    @include mixins.css-var(font-weight, clr-combobox-font-weight, inherit, variables.$clr-use-custom-properties);
+    font-size: combobox-variables.$clr-combobox-font-size;
+    font-weight: combobox-variables.$clr-combobox-font-weight;
 
     &.multi {
-      min-width: calc(min(combobox-variables.$clr-ng-multiselect-min-width, 100%));
-      padding-bottom: variables.$clr_baselineRem_0_125;
+      min-width: calc(min(combobox-variables.$clr-combobox-multi-min-width, 100%));
+      padding-bottom: mixins.baselinePx(3);
```

For refactoring ticket `padding-bottom` should be divisible by 2. 
```suggestion
      padding-bottom: tokens.$cds-global-space-3;
```

---

Refactoring ticket will be handled separately during the refactoring phase.
There is definitely some reason behind this, but I do not intend to search for it now.